amethyst_rendy/src/plugins.rs

```diff
@@ -64,6 +64,17 @@ mod window {
         }
 
         /// Clear window with specified color every frame.
+        /// This function takes linear RGBA. You can convert rgba to linear rgba like so (where your_r, your_g, your_b, and your_g are all u32):
+        ///
+        /// ```no_run
+        /// use amethyst::renderer::palette::Srgba;
+        ///
+        /// let (r, g, b, a) = Srgba::new(your_r / 255., your_g / 255., your_b / 255., your_a)
+        ///  .into_linear()
+        ///  .into_components();
```

Comment:
```suggestion
        ///     .into_linear()
        ///     .into_components();
```
We use 4 spaces as indentation.